CHANGELOG.md

```diff
@@ -168,6 +168,8 @@
   [#10247](https://github.com/Kong/kong/pull/10247)
 - Fix an issue where 'X-Kong-Upstream-Status' cannot be emitted when response is buffered.
   [#10056](https://github.com/Kong/kong/pull/10056)
+- Fix an issue where control plane does not downgrade config for `aws_lambda` and `zipkin` for older version of data planes.
```

Comment:
Because 3.2.0 has been released, should we move this entry to `unreleased` section?

Reply:
Unreleased means the next released version. In this case we should change the release number to `3.2.1` and mention "This version has been retracted because of known issues, please use Kong 3.2.1 instead" in the `3.2.0` section.

Reply:
I think we can keep this as-is for now. We need to fix the Changelog before 3.2.1 release anyway and best place to do this is in a separate PR.